backend/corpora/common/utils/cxg_generation_utils.py

```diff
@@ -113,69 +113,58 @@ def create_ndarray_array(ndarray_name, ndarray):
     tiledb.consolidate(ndarray_name, ctx=ctx)
 
 
-def convert_matrix_to_cxg_array(
-    matrix_name, matrix, encode_as_sparse_array, ctx, column_shift_for_sparse_encoding=None
-):
+def convert_matrix_to_cxg_array(matrix_name, matrix, encode_as_sparse_array, ctx):
     """
     Converts a numpy array matrix into a TileDB SparseArray of DenseArray based on whether `encode_as_sparse_array`
     is true or not. Note that when the matrix is encoded as a SparseArray, it only writes the values that are
     nonzero. This means that if you count the number of elements in the SparseArray, it will not equal the total
     number of elements in the matrix, only the number of nonzero elements.
-
-    Furthermore, if the `column_shift_for_sparse_encoding` matrix is not None, this function will subtract the sparse
-    encoding from the original given matrix and as previously stated, only write the nonzero values to the TileDB
-    SparseArray.
     """
 
-    def create_matrix_array(matrix_name, number_of_rows, number_of_columns, encode_as_sparse_array):
-        filters = tiledb.FilterList([tiledb.ZstdFilter()])
-        attrs = [tiledb.Attr(dtype=np.float32, filters=filters)]
-        if encode_as_sparse_array:
-            domain = tiledb.Domain(
-                tiledb.Dim(name="obs", domain=(0, number_of_rows - 1), tile=min(number_of_rows, 512), dtype=np.uint32),
-                tiledb.Dim(
-                    name="var", domain=(0, number_of_columns - 1), tile=min(number_of_columns, 2048), dtype=np.uint32
-                ),
-            )
-        else:
-            domain = tiledb.Domain(
-                tiledb.Dim(name="obs", domain=(0, number_of_rows - 1), tile=min(number_of_rows, 50), dtype=np.uint32),
-                tiledb.Dim(
-                    name="var", domain=(0, number_of_columns - 1), tile=min(number_of_columns, 100), dtype=np.uint32
-                ),
-            )
+    def create_matrix_array(matrix_name, number_of_rows, number_of_columns, encode_as_sparse_array, compression=22):
```

Comment:
Maybe add a comment with a link to [CXG schema optimization](https://docs.google.com/document/d/1Oxwua0_TsBWzgHSvc19OT8Lcf2jt3uGQxAZHN0ZGbEM/edit#heading=h.ftwmv4wub7jq) so that future engineers know from where these numbers originated? (256, 2048, 128000)